adoptedStyleSheets.js

```diff
@@ -70,7 +71,22 @@
       location.body ? location = location.body : null;
       clone[constructed] = location;  
       sheet[node]._adopters.push({ location, clone });
-      location.appendChild(clone);
+      if (location instanceof ShadowRoot) {
+        if (!location[styles]) {
+          location[styles] = document.createElement('head');
```

Comment:
This isn't a valid use of `HTMLHeadElement`, the only permitted location is as the [first child of an HTML tag as detailed here]*(https://developer.mozilla.org/en-US/docs/Web/HTML/Element/head). I'm happy to put the style elements inside a wrapper of some sort, but it should be either a div or a custom element. Perhaps a custom element that enforces proper ordering?

Reply:
Yeah, I'll change it to div or something else